### PR TITLE
Support get log level dynamically

### PR DESCRIPTION
**Pull request by tanshanshan**
**What this PR does / why we need it**:

I see there are set log level dynamically in PR #63777 #64601 
I think it is helpful for getting log level dynamically

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
 when you want to get glog level , you can send a GET request like `curl -X GET http://127.0.0.1:8080/debug/flags/v` .
```

### CODE REVIEW

File: staging/src/k8s.io/apiserver/pkg/util/logs/logs.go [hunk 1]
```diff
@@ -69,8 +69,12 @@ func NewLogger(prefix string) *log.Logger {
 	return log.New(GlogWriter{}, prefix, 0)
 }
 
-// GlogSetter is a setter to set glog level.
-func GlogSetter(val string) (string, error) {
+// GlogSetterOrGetter is to used to set glog level or get glog level .
+func GlogSetterOrGetter(val string) (string, error) {
+	if len(val) == 0 {
+		currentLevel := flag.Lookup("v").Value.(flag.Getter).Get().(glog.Level)
```
Comment (hzxuzhonghu) [inline staging/src/k8s.io/apiserver/pkg/util/logs/logs.go]: We can only get log level set by flag. But not that set by this handler. So I think it make no sense.

File: staging/src/k8s.io/apiserver/pkg/util/logs/logs.go [hunk 2]
```diff
@@ -69,8 +69,12 @@ func NewLogger(prefix string) *log.Logger {
 	return log.New(GlogWriter{}, prefix, 0)
 }
 
-// GlogSetter is a setter to set glog level.
-func GlogSetter(val string) (string, error) {
+// GlogSetterOrGetter is to used to set glog level or get glog level .
+func GlogSetterOrGetter(val string) (string, error) {
```
Comment (liggitt) [inline staging/src/k8s.io/apiserver/pkg/util/logs/logs.go]: mixing responsibilities in a single function like this isn't good... setting a flag to an empty string could be a reasonable thing to do. using it as a signal we want a get instead isn't great

File: staging/src/k8s.io/apiserver/pkg/util/logs/logs.go [hunk 3]
```diff
@@ -69,8 +69,12 @@ func NewLogger(prefix string) *log.Logger {
...
+	if len(val) == 0 {
+		currentLevel := flag.Lookup("v").Value.(flag.Getter).Get().(glog.Level)
+		return fmt.Sprintf("successfully get glog.logging.verbosity: %v", currentLevel), nil
```
Comment (liggitt) [inline staging/src/k8s.io/apiserver/pkg/util/logs/logs.go]: what's the use case for exposing this via a human-readable message?

**Pull request closed without merging**